web/conf/rhn_web.conf

```diff
@@ -63,7 +63,7 @@ web.version = 5.1.0
 # the version of Uyuni to show at the WebUI, it will be prepended
 # to web.version as version for the SPECs, when building them
 # for Uyuni
-web.version.uyuni = 2025.05
+web.version.uyuni = 2025.06
```

Comment:
Why `.06` ? We have at least `.07`

Reply:
Because I had open the old Uyuni card with version 2025.06 that should be renamed to the real release version (and I didn't connect the brain)

Reply:
@mcalmer I pushed 2025.07 now